intranet/apps/bus/migrations/0005_alter_route_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.4 on 2021-06-12 18:04
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bus', '0004_auto_20180117_1232'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='route',
+            options={'ordering': ['route_name']},
+        ),
+    ]
```

Comment:
Seems strange that there is a migration when `bus/models.py` doesn't seem to have changed.

Reply:
The migration is from a previous change, I forgot to run makemigrations when doing https://github.com/tjcsl/ion/commit/900edd5aa848148fdf92551dadb3da0f13773c57#diff-488cc7ad7f58d5a2e8f027e857ea887909a1b0df6c04dce0cb76fc0cabf46e15